gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/ReporterServiceProxyImpl.java

```diff
@@ -0,0 +1,146 @@
+    }
+
+    @Override
+    public Single<Reporter> create(String domain, NewReporter newReporter, User principal) {
+        return reporterService.create(domain, newReporter, principal);
+    }
+
+    @Override
+    public Single<Reporter> update(String domain, String id, UpdateReporter updateReporter, User principal) {
+        return reporterService.findById(id)
+                .switchIfEmpty(Single.error(new ReporterNotFoundException(id)))
+                .flatMap(oldReporter -> updateSensitiveData(updateReporter, oldReporter))
+                .flatMap(updateReporter1 -> reporterService.update(domain, id, updateReporter1, principal));
+    }
+
+    @Override
+    public Completable delete(String reporterId, User principal) {
+        return reporterService.delete(reporterId, principal);
+    }
+
+    private Single<Reporter> filterSensitiveData(Reporter reporter) {
+        return reporterPluginService.getSchema(reporter.getType())
+                .switchIfEmpty(Single.error(new ReporterPluginSchemaNotFoundException(reporter.getType())))
+                .map(schema -> {
+                    JsonNode configurationNode = objectMapper.readTree(reporter.getConfiguration());
+                    JsonNode schemaNode = objectMapper.readTree(schema);
+                    if (schemaNode.has("properties")) {
+                        schemaNode.get("properties").fields().forEachRemaining(entry -> {
+                            if (entry.getValue().has("sensitive") &&
+                                    entry.getValue().get("sensitive").asBoolean()) {
+                                ((ObjectNode) configurationNode).put(entry.getKey(), SENSITIVE_VALUE);
+                            }
+                        });
+                        reporter.setConfiguration(configurationNode.toString());
+                    }
+                    return reporter;
+                });
+    }
+
+    private Single<UpdateReporter> updateSensitiveData(UpdateReporter updateReporter, Reporter oldReporter) {
+        return reporterPluginService.getSchema(oldReporter.getType())
+                .switchIfEmpty(Single.error(new ReporterPluginSchemaNotFoundException(oldReporter.getType())))
+                .map(schema -> {
+                    JsonNode updatedConfigurationNode = objectMapper.readTree(updateReporter.getConfiguration());
+                    JsonNode oldConfigurationNode = objectMapper.readTree(oldReporter.getConfiguration());
+                    JsonNode schemaNode = objectMapper.readTree(schema);
+                    if (schemaNode.has("properties")) {
+                        schemaNode.get("properties").fields().forEachRemaining(entry -> {
+                            if (entry.getValue().has("sensitive") &&
+                                    entry.getValue().get("sensitive").asBoolean()) {
+                                if (SENSITIVE_VALUE.equals(updatedConfigurationNode.get(entry.getKey()).asText())) {
```

Comment:
Hi, in order to limit the risk of overwriting in case of false manipulation (deletion of a * for example in the ui) is it possible at the front end to make the displayed value of the field in the form not be the real value (null value?). And check here that the value is null or only space maybe.